lizmap/modules/lizmap/lib/Project/QgisProject.php

```diff
@@ -1259,6 +1278,21 @@ protected function readWMSInformation($qgsLoad)
         );
     }
 
+    /**
+     * Read  the last modified date of the QGS file.
+     *
+     * @param \SimpleXMLElement $xml
+     *
+     * @return string
+     */
+    protected function readLastSaveDateTime($xml)
+    {
+        $qgisRoot = $xml->xpath('//qgis');
+        $qgisRootZero = $qgisRoot[0];
+
+        return (string) $qgisRootZero->attributes()->saveDateTime;
+    }
```

Comment:
This would consume a lot of RAM, we need to check how to solve this. Let-s check other functions in the same file too.